Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -138,7 +138,7 @@ def request(self, path=None, params=None, headers=None, authenticated=None, pagi
         if authenticated is False:
             auth = None
         if authenticated and not auth:
-            raise self.Exception('Request requires authentication, none provided')
+            raise RuntimeError('Request requires authentication, none provided')
```

Comment:
Why do we make this change? I don't find it in commit message.

Reply:
Will add it. We don't actually have a `self.Exception` in this class. The code was moved from a class that did have this.